config/config_test.go

```diff
@@ -67,6 +67,46 @@ func TestBlacklist(t *testing.T) {
 	}
 }
 
+func TestArgsBlacklist(t *testing.T) {
+	customArgsBlacklist := []string{
+		"consul_token",
+		"dd_password",
+		"blocked_from_yaml",
+	}
+
+	defaultRegexs := CompileStringsToRegex(defaultArgsBlacklist)
+	customRegexs := CompileStringsToRegex(customArgsBlacklist)
+	mergedRegexs := append(defaultRegexs, customRegexs...)
+	t.Log("default regexp", defaultRegexs)
+	t.Log("custom regexp", customRegexs)
+	t.Log("merged regexp", mergedRegexs)
+
+	cases := []struct {
+		cmdline       []string
+		parsedCmdline []string
+	}{
+		{[]string{"agent", "-password", "1234"}, []string{"agent", "-password", "********"}},
+		{[]string{"agent", "--password", "1234"}, []string{"agent", "--password", "********"}},
+		{[]string{"agent", "-password=1234"}, []string{"agent", "-password=********"}},
+		{[]string{"agent", "--password=1234"}, []string{"agent", "--password=********"}},
+		{[]string{"spidly", "-debug_port=2043"}, []string{"spidly", "-debug_port=2043"}},
```

Comment:
Lets split these no-change cases into it's own test function, so its a bit clearer for which cases it should be modified, and which should not.